platform/lib/common/filteredPage.js

```diff
@@ -148,13 +160,15 @@ class FilteredPage {
     // Rewrite the active state (which is websites per default) to
     // the current active format
     const activeFormat = this._dom('.ap-m-format-toggle-selected');
-    if (!activeFormat.html()) {
-      console.error('doc did not specify an active format');
+    if (activeFormat.length == 0) {
+      throw new Error('Page has no active format.');
```

Comment:
I don't like throwing an exception here, I it happens in prod the user gets a 502, but it's something we can recover easily from.

Reply:
Ah you are right. I only had the implementation for local in mind, the error is catched there. Will revert.